src/app/bucket/bucket.component.html

```diff
@@ -24,27 +24,33 @@
     <mat-list-item *ngFor="let objective of bucket!.objectives">
     <app-objective
         [objective]="objective" [unit]="unit"
+        [sum]="cumilativeSum"
+        (cumiliativeSum)=setCumilativeSum($event)
+        [bucketAllocationLimit]="bucketAllocationLimit()"
         [unallocatedTime]="unallocatedTime"
         [isEditingEnabled]="isEditingEnabled"
         [isReorderingEnabled]="showOrderButtons"
         [otherBuckets]="otherBuckets"
         (onMoveBucket)="moveObjective($event[0], $event[1], $event[2])"
         (onDelete)="deleteObjective($event)"
-        (onChanged)="onObjectiveChanged($event[0], $event[1])"></app-objective>
+        (onChanged)="onObjectiveChanged($event[0], $event[1])" class="width"></app-objective>
     </mat-list-item>
 </mat-list>
 <mat-list dense *ngIf="showOrderButtons" cdkDropList (cdkDropListDropped)="reorderDrop($event)" class="app-mat-list">
     <mat-list-item *ngFor="let objective of bucket!.objectives" cdkDrag>
         <i class="material-icons reorder-icon" cdkDragHandle>reorder</i>
         <app-objective
             [objective]="objective" [unit]="unit"
+            (cumiliativeSum)=setCumilativeSum($event)
```

Comment:
As noted elsewhere, we shouldn't call methods with side effects in the template.